projects/storyteller/webapp/src/app/core/services/firebase.service.ts

```diff
@@ -24,7 +25,8 @@ export class FirebaseService {
       this.angularFireAuth.authState.subscribe(userData => {
       this.isOnline = !!userData;
       });
-    this.protobin = this.db.collection('storyteller');
+      this.protobin = this.db.collection(`/storyteller/data/user/valerii.fedorenko.ua@gmail.com/story`);
+      this.protobin_screenshots = this.db.collection('/storyteller/data/user/valerii.fedorenko.ua@gmail.com/screenshot');
```

Comment:
This string is too long. Please don't exceed 100 chars per line.
It's related to html code too. Can you find out which files have this issue too?